iidm/iidm-test/src/main/java/com/powsybl/iidm/network/test/LimitsTestFactory.java

```diff
@@ -0,0 +1,20 @@
+package com.powsybl.iidm.network.test;
+
+import com.powsybl.iidm.network.CurrentLimitsAdder;
+
+public final class LimitsTestFactory {
+
+    private LimitsTestFactory() {
+    }
+
+    public static void createLimits(CurrentLimitsAdder limitsAdder) {
```

Comment:
Is it really useful

Reply:
Since it is only used in order to test limits per season, i moved it as an utility class there